department-of-veterans-affairs/caseflow
# Release to all attorneys & access control cleanup

Resolves #5547

### Acceptance Criteria 
- All attorneys have access to the queue where they can view tasks
- We can roll out to judges with a separate feature toggle

### Testing Plan
- `FeatureToggle.disable!(:queue_welcome_gate)`
- Log in as attorney user
- Successfully view `/queue` page
- Log in as judge user
- `/queue` page should result in access error
- `FeatureToggle.enable!(:judge_queue)`
- Successfully view `/queue` page
- `FeatureToggle.enable!(:case_search_home_page)`
- Navigate to `/`, observe that we're redirected to the Queue home page
